kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -1128,4 +1170,21 @@ public void setOauthTokenProvider(OAuthTokenProvider oauthTokenProvider) {
   public void setCustomHeaders(Map<String, String> customHeaders) {
     this.customHeaders = customHeaders;
   }
+
+  public List<NamedContext> getContexts() {
```

Comment:
I would appreciate it if you could add some Javadoc to publicly available methods introduced in this PR.